docs/data/date-pickers/date-picker/date-picker.md

```diff
@@ -98,13 +98,22 @@ You can take advantage of the [PickersDay](/x/api/date-pickers/pickers-day/) com
 
 You can customize the month layout with some props.
 
+### Show additional days
+
 To shows all days of displayed weeks, included those outside of the current month, use `showDaysOutsideCurrentMonth`.
 
 By default, only weeks of the current month are displayed, but you can provide a total number of week to display with `fixedWeekNumber` prop.
 This value is usually set to `6` for Gregorian calendars, because months display can vary between 4 and 6 weeks.
 
 {{"demo": "CustomMonthLayout.js"}}
 
+### Add week number
```

Comment:
This is also available on the date time pickers and the date range pickers
How can we make that clear ? (same for `showDaysOutsideCurrentMonth` by the way).

Should we add a small section on those pages saying that the calendar customization of the date picker also apply to them ?
Or move this content in a `DateCalendar` / `DayCalendar` page ?

Reply:
I would say on the Date Calendar page ultimately with a small note on the Date Picker and Date Time Picker page to make sure people find it.

But for now maybe on the Date Picker page with a small note on the Date Time Picker page.

If @joserodolfofreitas or @samuelsycamore have an opinion here, it's a topic that will happen more and more.

Reply:
Based on the mentioned examples, I think the Date Calendar sounds like a good page to host these props.
But I feel we're adding a layer of complexity and it can maybe start getting confusing if we spread these high-level properties in different subcomponent pages.

We briefly discussed a slightly different variation of this idea, but maybe we could consider a shared properties file and then re-print the documentation on every main component page (with the properties applicable to that component). We may not need to have an "original link" to host them.